src/java.naming/share/classes/javax/naming/Context.java

```diff
@@ -1082,6 +1082,6 @@ public Object removeFromEnvironment(String propName)
      *
      * @since 1.3
      */
-    @Deprecated
+    @Deprecated(since="16", forRemoval=true)
     String APPLET = "java.naming.applet";
 };
```

Comment:
Probably should be since="9" (the deprecation in JDK-8051422 pre-dates the enhanced deprecation work).

Reply:
Good point, since it was in fact deprecated in 9.

Reply:
yes - changed to since="9" this morning